src/main/groovy/uk/ac/ox/softeng/maurodatamapper/plugins/database/DatabaseDataModelImporterProviderServiceParameters.groovy

```diff
@@ -109,6 +109,30 @@ abstract class DatabaseDataModelImporterProviderServiceParameters<K extends Data
         ))
     Boolean databaseSSL
 
+    @ImportParameterConfig(
+        displayName = 'Detect Enumerations',
+        description = 'Whether to treat columns with small numbers of unique values as enumerations',
+        order = 1,
+        optional = true,
+        group = @ImportGroupConfig(
+                name = 'Configuration',
+                order = 2
+        )
+    )
+    Boolean detectEnumerations = false
+
+    @ImportParameterConfig(
+        displayName = 'Maximum Enumerations',
+        description = 'The maximum number of unique values to be interpreted as a defined enumeration',
+        order = 2,
+        optional = true,
+        group = @ImportGroupConfig(
+                name = 'Configuration',
+                order = 2
```

Comment:
Can we change the group to "Database Import Details" as per "Database Name(s)"

Reply:
Done